apps/balancer-tools/src/components/Tabs.tsx

```diff
@@ -61,7 +61,7 @@ export function Tabs<T extends string | undefined>({
         value={value}
         defaultValue={defaultValue}
       >
-        <TabsPrimitive.List className="flex shrink-0 flex-col border-b border-blue1 bg-blue3">
```

Comment:
idk why this existed before

<img width="349" alt="image" src="https://github.com/bleu-studio/balancer-tools/assets/63298341/c280cbb1-3e12-44d9-a42c-89ed5ff19d99">

Reply:
this screen shot is from poolsimulator but this border just adds this line which seems out of place